elasticsearch/README.md

```diff
@@ -88,6 +88,9 @@ helm install --name elasticsearch elastic/elasticsearch --version 7.0.1-alpha1 -
 | `protocol`                 | The protocol that will be used for the readinessProbe. Change this to `https` if you have `xpack.security.http.ssl.enabled` set                                                                                                                                                                                            | `http`                                                                                                                    |
 | `httpPort`                 | The http port that Kubernetes will use for the healthchecks and the service. If you change this you will also need to set [http.port](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-http.html#_settings) in `extraEnvs`                                                                          | `9200`                                                                                                                    |
 | `transportPort`            | The transport port that Kubernetes will use for the service. If you change this you will also need to set [transport port configuration](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-transport.html#_transport_settings) in `extraEnvs`                                                        | `9300`                                                                                                                    |
+| `service.type`             | Type of elasticsearch service. [Service Types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)                                                                                                                                                                         | `ClusterIP`                                                                                                                    |
+| `service.annotations`      | Annotations that Kubernetes will use for the service. This will configure load balancer if `service.type` is `LoadBalancer` (https://kubernetes.io/docs/concepts/services-networking/service/#ssl-support-on-aws)                                                                                                          | `{}`                                                                                                                    |
```

Comment:
Could you add an alias to the url to make it a bit shorter? Maybe adding it to [Annotations] would make the most sense